examples/src/go1_test.cpp

```diff
@@ -214,5 +214,9 @@ int main(int argc, char **argv)
                          wrench_legend_names);
     plotter.PlotConstraints();
 
+    Eigen::MatrixXd new_state(new_sol.state_result.rows()-1, new_sol.state_result.cols());
+    Eigen::MatrixXd euler_angles = quatToEulerZYX(new_sol.state_result.block(si->nh + si->ndh + 3, 0, 4, new_sol.state_result.cols()));
```

Comment:
This should be  galileo::legged::quat2EulerZYX

Reply:
go1.h already includes the legged namespace. The code compiles and runs correctly